Changes in pom

acerlife opened the pull request.

acerlife committed:
Changes in pom

VolodymyrMudryk commented on pom.xml:
```diff
@@ -8,7 +8,28 @@
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
```
this dependency is redundant, please use only `spring-boot-starter-web`

VolodymyrMudryk commented on pom.xml:
```diff
@@ -8,7 +8,28 @@
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-jooq</artifactId>
```
at this moment the latest version is 3.10.6 https://mvnrepository.com/artifact/org.jooq/jooq/3.10.6
please check which version in `spring-boot-starter-jooq`

VolodymyrMudryk reviewed (changes requested):
few comments
in additional please use more clear the commit name

acerlife committed:
Update pom.xml

Change jooq version. Rename spring-boot-starter-dependencies on spring-boot-starter-web

VolodymyrMudryk commented on pom.xml:
```diff
@@ -8,7 +8,48 @@
+                <groupId>org.jooq</groupId>
+                <artifactId>jooq</artifactId>
+                <version>3.10.6</version>
+                <type>pom</type>
```
why do you need this?
